packages/eslint-plugin-eds/src/index.ts

```diff
@@ -0,0 +1,11 @@
+import noHTags from './rules/no-h-tags';
+import noPTags from './rules/no-p-tags';
+
+const rules = {
+  'no-h-tags': noHTags,
+  'no-p-tags': noPTags,
+};
+
+module.exports = {
+  rules,
```

Comment:
Don't think anything more complex than this is necessary?

Reply:
Not unless you also want to export a recommended config (which the readme sez we do)

Reply:
Added a recommended config, lmk how it looks